mkdocs_awesome_pages_plugin/tests/navigation/test_meta.py

```diff
@@ -45,34 +50,68 @@ def setUp(self):
         super(TestMeta, self).setUp()
         self.options = Options(filename=".pages", collapse_single_pages=False, strict=True)
 
+    def _make_nav_meta(
+        self,
+        items: List[NavigationItem],
+        options: Options,
+        explicit_sections: Set[Section] = frozenset(),
+    ):
+        files = Files([])
+        config_dirs = set()
+
+        def populate_config_dirs(items: List[NavigationItem]):
+            for item in items:
+                if isinstance(item, Page) and not item.url.startswith("/"):
+                    config_dirs.add(
+                        (
+                            os.path.dirname(item.file.src_path),
+                            os.path.dirname(item.file.abs_src_path),
+                        )
+                    )
+                    files.append(item.file)
+                elif isinstance(item, Section):
+                    populate_config_dirs(item.children)
+
+        populate_config_dirs(items)
+
+        for config_dir, docs_dir in config_dirs:
```

Comment:
Something with `docs_dir` is off. Either the name is misleading or something on line 68 is wrong. I would argue the dirname of the absolute source path isn't always the same as the src_dir.